### ISSUE TITLE
libpng-deploy/ARCH=x86_64,NAME=libpng,OS=centos6,SITE=generic,VERSION=1.6.27 #14 failed

### ISSUE THREAD

**Issue by jenkinsSAGrid** [SouthAfricaDigitalScience/libpng-deploy#24]
Build 'libpng-deploy/ARCH=x86_64,NAME=libpng,OS=centos6,SITE=generic,VERSION=1.6.27' is failing!

Last 50 lines of build output:

```
[...truncated 26.90 KB...]
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking that AWK works... ok
checking if we need to force back C standard to C89... no
checking for ANSI C header files... (cached) yes
checking for an ANSI C-conforming const... yes
checking for size_t... yes
checking whether struct tm is in sys/time.h or time.h... time.h
checking for C/C++ restrict keyword... __restrict
checking for working strtod... yes
checking for memset... yes
checking for pow... no
checking for pow in -lm... yes
checking for clock_gettime... no
configure: WARNING: not building timepng
checking for zlibVersion in -lz... no
checking for /data/ci-build/generic/centos6/x86_64/zlib/1.2.8zlibVersion in -lz... no
configure: error: zlib not installed
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
No changes
- [Bruce Becker] c711d716f2ab418dfd60135f63632448258133ab - Minor Release Update

No changes
- [Bruce Becker] 64dd4c8447d873730a1f91b881d771044eabb791 - Update README with configuration

- [Bruce Becker] fb79d390aa05286fdef4e3c85bba72d055c0d5c0 - Force prepend /lib64 to . Close #23

[View full output](https://ci.sagrid.ac.za/job/libpng-deploy/ARCH=x86_64,NAME=libpng,OS=centos6,SITE=generic,VERSION=1.6.27/14/)

**Comment by brucellino** [comment]
the configure error was : 

```
gcc -o conftest -I/data/ci-build/generic/centos6/x86_64/zlib/1.2.8/include -I/data/ci-build/generic/centos6/x86_64/zlib/1.2.8/include -L/data/ci-build/generic/
centos6/x86_64/zlib/1.2.8/lib/ conftest.c -lz  -lm  >&5
conftest.c:34: error: expected identifier or '(' before '/' token
conftest.c:34:49: error: too many decimal points in number
conftest.c: In function 'main':
conftest.c:38: error: expected expression before '/' token
conftest.c:38:51: error: too many decimal points in number
```

It's not clear to me what the kssue is.

**Comment by brucellino** [comment]
another clue : 
```
/data/ci-build/generic/centos6/x86_64/zlib/1.2.8/lib//libz.so: undefined reference to `memcpy@GLIBC_2.14'
```

**Comment by brucellino** [comment]
Closed in 8ff711a
